pkg/driver/controller.go

```diff
@@ -440,6 +440,9 @@ func (d *controllerService) CreateSnapshot(ctx context.Context, req *csi.CreateS
 	opts := &cloud.SnapshotOptions{
 		Tags: map[string]string{cloud.SnapshotNameTagKey: snapshotName},
 	}
+	for k, v := range d.driverOptions.extraVolumeTags {
```

Comment:
Do you also want to add cluster id tag, similar to how it's done for CreateVolume?

Reply:
Added this and an additional unit test to cover this case.